nifi-nar-bundles/nifi-hubspot-bundle/nifi-hubspot-processors/src/main/resources/docs/org.apache.nifi.processors.hubspot.GetHubSpot/additionalDetails.html

```diff
@@ -32,5 +32,13 @@ <h2>Incremental Loading</h2>
     last run time of the processor are processed. The processor state can be reset in the context menu. The incremental loading
     is based on the objects last modified time.
 </p>
+<p>
+    There is no deletion tolerance in the current implementation. Some objects may be omitted if any object is deleted between fetching two pages.
```

Comment:
I think it rather belongs to the `Paging` section so I would move it there.
Also, I would mention that the deletion tolerance issue is due to the limitations of the HubSpot API. Something like this:
```suggestion
<p>
    Due to the page handling mechanism of the HubSpot API, parallel deletions are not supported. Some objects may be omitted if any object is deleted between fetching two pages.
```